File: app/lessondz3v3/src/main/java/com/example/lessondz3v3/MainActivity.java
package com.example.lessondz3v3;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    private RecyclerView rvPerehot;
    private ArrayList<String> perehotListMain = new ArrayList<>();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        rvPerehot = findViewById(R.id.rv_btn_list);

        Perehodnoe();

        PerehotAdapter adapter = new PerehotAdapter(perehotListMain);
        rvPerehot.setAdapter(adapter);

    }

    private void Perehodnoe() {
        perehotListMain.add("Перейти: ");

    }
}

File: app/lessondz3v7/src/main/java/com/example/lessondz3v7/MainActivity.kt
package com.example.lessondz3v7

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.lessondz3v7.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}